shared/settings/nav/index.desktop.js

```diff
@@ -0,0 +1,72 @@
+// @flow
+import React, {Component} from 'react'
+import {globalStyles, globalColors, globalMargins} from '../../styles'
+import {Box, Badge, ClickableBox, Text} from '../../common-adapters'
+
+import type {Props, BannerType, SettingsItem as SettingsItemType} from './index'
+
+function Banner ({element, type}: {element: React$Element<*>, type: BannerType}) {
+  return (
+    <Box style={{...commonBannerStyle, ...variantBannerStyle[type]}}>
```

Comment:
This is much simpler, it doesn't even make assumption to what could be inside, centering is the caller's responsibilty this just gives it a place to put an element and the background.